src/custom/ecospheres/store/IndicatorStore.ts

```diff
@@ -33,7 +33,7 @@ export const useIndicatorStore = defineStore('indicator', {
         tag,
         ...extraArgs
       })
-      this.indicators = results.data
+      this.indicators = results.data as Indicator[]
```

Comment:
Force typing to make the translation between a generic dataset from the API and an indicator.